doc/classes/int.xml

```diff
@@ -79,7 +77,7 @@
 			<return type="bool" />
 			<param index="0" name="right" type="float" />
 			<description>
-				Returns [code]true[/code] if this [int] is not equivalent to the given [float].
+				Returns [code]true[/code] if the integer is not equivalent to the [float].
```

Comment:
Here you replace a reference to `[int]` with the word "integer", but in other operators you do the opposite, and replace the existing word "integer" with `the [int]`. Which should it be? :)